libraries/AC_WPNav/AC_Circle.h

```diff
@@ -134,7 +134,8 @@ class AC_Circle
     };
 
     // parameters
-    AP_Float    _radius;        // maximum horizontal speed in cm/s during missions
+    AP_Float    _radius;        // radius of circle in cm
```

Comment:
```suggestion
    float _radius;        // radius of circle in cm
```
should also be down with the internal variables